<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SpriteEditor</class>
 <widget class="QDialog" name="SpriteEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>677</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sprite Editor</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Sprite Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="spriteNameInput"/>
         </item>
         <item>
          <widget class="QPushButton" name="selectSpritesheetButton">
           <property name="text">
            <string>Select Spritesheet</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QToolButton" name="autoFrameSelectTool">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/autoframeselect.png</normaloff>:/Icons/autoframeselect.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>toolButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="frameSelectTool">
           <property name="toolTip">
            <string>Frame Selection Tool</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/frameselect.png</normaloff>:/Icons/frameselect.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="popupMode">
            <enum>QToolButton::DelayedPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <attribute name="buttonGroup">
            <string>toolButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="renderSpotTool">
           <property name="toolTip">
            <string>Render Spot Tool</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/renderspot.png</normaloff>:/Icons/renderspot.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>toolButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="addHotspotTool">
           <property name="toolTip">
            <string>Add Hotspot Tool</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/hotspot.png</normaloff>:/Icons/hotspot.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
           <attribute name="buttonGroup">
            <string>toolButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="pointerTool">
           <property name="toolTip">
            <string>Pointer Tool</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/selector.png</normaloff>:/Icons/selector.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string>toolButtons</string>
           </attribute>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="zoomLevelLabel">
           <property name="text">
            <string>100% Zoom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="zoominTool">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/zoomin.png</normaloff>:/Icons/zoomin.png</iconset>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="zoomoutTool">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../Icons.qrc">
             <normaloff>:/Icons/zoomout.png</normaloff>:/Icons/zoomout.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGraphicsView" name="imageView">
         <property name="dragMode">
          <enum>QGraphicsView::NoDrag</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Animations</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="animationList">
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::MoveAction</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addAnimationButton">
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Add Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="editAnimationButton">
          <property name="text">
           <string>Edit Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="previewButton">
          <property name="text">
           <string>Preview Animation</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deleteAnimationButton">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Delete Animation</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="addFrameButton">
       <property name="text">
        <string>Add Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteFrameButton">
       <property name="text">
        <string>Delete Frame</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Frame Delay:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="frameDelayInput">
       <property name="suffix">
        <string> ms</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="frameList">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="dragDropMode">
      <enum>QAbstractItemView::NoDragDrop</enum>
     </property>
     <property name="defaultDropAction">
      <enum>Qt::IgnoreAction</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="movement">
      <enum>QListView::Static</enum>
     </property>
     <property name="flow">
      <enum>QListView::LeftToRight</enum>
     </property>
     <property name="viewMode">
      <enum>QListView::IconMode</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SpriteEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SpriteEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>667</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>animationList</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>editAnimationButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>602</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="toolButtons"/>
 </buttongroups>
</ui>
